concepts/0003-protocols/state-details.md

```diff
@@ -1,80 +1,39 @@
-# State Details and State Machines
+### State Details and State Machines
 
-Some protocols have only one sequence of states to manage. But in many
-protocols, different roles perceive the interaction differently. Each
-perspective needs to be described with care. This rigor has been neglected
-in many early efforts at protocol definition, and its omission is a big
-miss. Analyzing all possible states and events for all roles leads to
-robustness; skipping the analysis leads to fragility.
+While some protocols have only one sequence of states to manage, in most
+different roles perceive the interaction differently. The sequence of states
+for each role needs to be described with care in the RFC.
 
-### State machines
+#### State machines
 
-By convention, our community describes state and sequence rules using the
+By convention, protocol state and sequence rules are described using the
 concept of state machines, and we encourage developers who implement
 protocols to build them that way.
 
 Among other benefits, this helps with error handling: when one agent
-sends a [`problem-report`](../../features/0035-report-problem/README.md)
+sends a [`report-problem`](../../features/0035-report-problem/README.md)
```

Comment:
The protocol is named "report problem"; the message type is "problem-report".

Reply:
Added "message" to the references.